CHANGELOG.md

```diff
@@ -16,6 +16,13 @@
 
 ---
 
+## v4.0.1
+
+- 🐛 let the backend to most of the tags sanitization ([#1252](https://github.com/DataDog/browser-sdk/pull/1252))
+- 🐛 remove child with action-names innerText instead of replacing them ([#1251](https://github.com/DataDog/browser-sdk/pull/1251))
```

Comment:
should we update breaking changes content of 4.0.0 to reflect those changes?

Reply:
Yes good idea